image/src/teachable-mobilenet.ts

```diff
@@ -376,6 +376,10 @@ export class TeachableMobileNet extends CustomMobileNet {
         return this._metadata.modelName;
     }
 
+    public stopTraining() {
```

Comment:
I think this is worth adding a test for.
I think this also should return a promise. My understanding is that `stopTraining` is asynchronous and it could be important for the user to understand when this has resolved.

Reply:
another thought, can this be resumed? I don't believe it cant. If it can't I think `cancelTraining` is more fitting. If it can then we should add `resumeTraining`

Reply:
can't be resumed. added a test!